### ISSUE TITLE
[validation] Element figcaption not allowed as child of element div.

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#166]
URL: https://openreferraluk.org/

Element figcaption not allowed as child of element div.

**Incorrect**
```
<figure role="figure" class="figure-block">
  <div class="page-container">
    <div class="format">
      <blockquote>The services out there can change peoples' lives. But if we don't have accurate and reliable information, then we're working with one hand tied behind our back. We have a limited chance of helping them.</blockquote>. 
      <figcaption>– Key worker in Blackburn</figcaption>
    </div>
  </div>
</figure>
```

**Correct**
```
<figure role="figure" class="figure-block">
  <blockquote>The services out there can change peoples' lives. But if we don't have accurate and reliable information, then we're working with one hand tied behind our back. We have a limited chance of helping them.</blockquote>. 
  <figcaption>– Key worker in Blackburn</figcaption>
</figure>
```

The additional container divs for formating would need to enclose the figure, maybe adding a `<section>` as the outer most element with a white background.

**Comment by UsmanAfzal** [comment]
Checked and approved
